RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -48,21 +48,14 @@
     jetAlgorithm = cms.string("Kt"),
     rParam       = cms.double(0.4)
 )
+
 kt4PFJetsForRho.src = cms.InputTag('particleFlow')
 kt4PFJetsForRho.doAreaFastjet = cms.bool(True)
 kt4PFJetsForRho.jetPtMin      = cms.double(0.0)
 kt4PFJetsForRho.GhostArea     = cms.double(0.005)
 
-hiFJRhoProducer = cms.EDProducer('HiFJRhoProducer',
-                                 jetSource = cms.InputTag('kt4PFJetsForRho'),
-                                 nExcl = cms.int32(2),
-                                 etaMaxExcl = cms.double(2.),
-                                 ptMinExcl = cms.double(20.),
-                                 nExcl2 = cms.int32(1),
-                                 etaMaxExcl2 = cms.double(3.),
-                                 ptMinExcl2 = cms.double(20.),
-                                 etaRanges = cms.vdouble(-5., -3., -2.1, -1.3, 1.3, 2.1, 3., 5.)
-)
+from RecoHI.HiJetAlgos.hiFJRhoProducer import hiFJRhoProducer
+hiFJRhoProducer.etaRanges = cms.vdouble(-5., -3., -2.1, -1.3, 1.3, 2.1, 3., 5.)
```

Comment:
Modifying modules parameters outside their own configuration file can be dangerous, as those changes can be propagated also to configurations in which you don't want to do so. As you are doing it now, the etaRanges are also going to be modified for the ```pA_2016``` era, while they were previously  the "default" 
 ```etaRanges = cms.vdouble(-5., -3., -2., -1.5, -1., 1., 1.5, 2., 3., 5.)```

My suggestion in order to retain the previous etaRanges for the ```pA_2016``` era and update them for the AA runs is the following:
- Remove the modification here (and if I am not wrong you don't even need to import ```hiFJRhoProducer``` here)
- Update the default ```etaRanges``` to the new ones in ```RecoHI/HiJetAlgos/python/hiFJRhoProducer.py```
- Add a modifier in ```RecoHI/HiJetAlgos/python/hiFJRhoProducer.py``` for ```pA_2016``` and put back with it the previous ```etaRanges``` only for that era

As such the modifier is applied in the file which defines the hiFJRhoProducer module, and it will be far much easier to maintain (and the ```pA_2016```  reconstruction won't be affected)